tests/acceptance-tests/src/acceptance/http_closed_test.go

```diff
@@ -31,13 +31,13 @@ var _ = Describe("Http is closed", func() {
 
 		uri := appName + "." + config.AppsDomain + ":80"
 		_, err := net.DialTimeout("tcp", uri, CONNECTION_TIMEOUT)
-		Expect(err.(net.Error).Timeout()).To(BeTrue())
+		Expect(err).To(HaveOccurred())
```

Comment:
We actually can't use type assertion only because it will return an error when one of the values is nil: https://godoc.org/github.com/onsi/gomega#BeAssignableToTypeOf

I added `Expect(err).ToNot(BeNil())` before type assertion.